src/Adafruit_SPIFlashBase.h

```diff
@@ -47,7 +47,7 @@ class Adafruit_SPIFlashBase {
   Adafruit_SPIFlashBase(Adafruit_FlashTransport *transport);
   ~Adafruit_SPIFlashBase() {}
 
-  bool begin(SPIFlash_Device_t const *flash_dev = NULL);
+  bool begin(SPIFlash_Device_t const *flash_devs = NULL, int count = 1);
```

Comment:
let change the type of count to size_t since we don't need negative value. GCC can complain when we do compare signed vs unsigned